### ISSUE TITLE
dbPool has argument "user" not "username"

### ISSUE THREAD

**Issue by koseri** [rstudio/pool#13]
I tried to follow the example a little bit and ran into trouble:

```
pool <- dbPool(
  drv = "PostgreSQL",
  host = db_host,
  dbname = db,
  username = 'foo',
  password = 'bar'
)
```
Produces
```
Error in postgresqlNewConnection(drv, ...) : 
  unused argument (username = "foo")
```
while
```
pool <- dbPool(
  drv = "PostgreSQL",
  host = db_host,
  dbname = db,
  user = 'foo',
  password = 'bar'
)
```
produces something a little more expected:
```
Error in postgresqlNewConnection(drv, ...) : 
  RS-DBI driver: (could not connect foo@192.1.22.33 on dbname "some_db")
```

**Comment by bborgesr** [comment]
`dbPool` does not have a `user` or `username` argument. It has a `drv` argument and a `...` argument. Whatever you pass through `...` should match what the `drv` needs. While the `RySQL` driver has a `username` arg, the `PostgreSQL` diver has a `user` arg. No issue there.

As for the the error you got. Either `some_db` is not a a valid database in that location, or the username is wrong, or the password in wrong, etc... The driver is not meant to connect you if you pass in the wrong arguments.

**Comment by koseri** [comment]
Got it. I didn't notice it because the examples only showed MySQL and because I don't usually have to use the user argument (as you might infer from my issue #14). I was just trying to see if I could get the examples working as close to as they were before I started messing with them... controlling for the fact that I don't use MySQL.

I only included the example of the expected error to show that I could get past the "unused argument" error message into an error that is "expected" -- when trying to access a bogus database. Sorry for the confusion on that.

Thanks!
